applications/CompressiblePotentialFlowApplication/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ set( KRATOS_COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION_SOURCES
 )
 
 # define library Kratos which defines the basic python interface
-add_library(KratosCompressiblePotentialFlowApplication SHARED ${KRATOS_COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION_SOURCES})
-target_link_libraries(KratosCompressiblePotentialFlowApplication KratosCore )
+pybind11_add_module(KratosCompressiblePotentialFlowApplication MODULE ${KRATOS_COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION_SOURCES})
```

Comment:
Here you should add `THIN_LTO`, see [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/FluidDynamicsApplication/CMakeLists.txt#L91)